x-pack/test/functional/config.base.js

```diff
@@ -480,6 +480,20 @@ export default async function ({ readConfigFile }) {
           },
         },
 
+        security_user: {
```

Comment:
My only other comment is:

The term `security_user` is a bit too generic in my opinion, given that we have a Security solution.
Maybe `cluster_security_manager` would be a better name for this role.
WDYT?

Reply:
Fixed.